### DISCUSSION TITLE
How to download single files without a folder

### DISCUSSION THREAD

**Discussion by doctorase** [rogerfar/rdt-client#467] [Q&A]
The client downloads the single files into a folder with the same name as the file, including the file extension. For example the file TVSHOW1.mkv is downloaded into a generated folder named TVSHOW1.mkv.

I guess this is due to a misconfiguration, also when I use sonarr to add the download, as it does it inside this folder, then it is not able to import the file from sonarr. 

How can I configure to fix this?

Thanks!

**Comment by rogerfar** [comment]
I'm not sure what you mean, it should work just out of the box. There isn't really much to configure on that end. 

It also doesn't really matter because RDT tells Sonarr / Radarr where it can find the file, so unless there is a disconnect there, it should work.

**Comment by doctorase** [comment]
Even without sonarr and downloading the file with a normal magnet link, it generates a folder with the same name of the file.
Exemple of downloaded file:
[https://imgur.com/iWobrdQ](url)
Generates this folder with the file inside
[https://i.imgur.com/UJZfBW4.png](url)

Normally all clients download the file without generating a new folder, unless it is a set of files in the same magnet link.

**Comment by rogerfar** [comment]
I understand your point, but is this a problem down the line with radarr/sonarr, or is it a general observance and annoyance?

**Comment by doctorase** [comment]
It's a general observance and annoyance. I don't know if is normal with rdt-client because all other torrent clients download the file without a folder, or you can choose it.

**Comment by pavichokche** [comment]
I also wanted to voice my opinion that this would be a great toggle to add. Qbittorrent already supports the option to not create a subfolder for a torrent's contents, both globally and per torrent when adding with manual settings, so perhaps it's available as a toggle in the API as well...

**Comment by harun911** [comment]
I would love to have the option too, to get the files as is, without a folder being created where they are put in.
